ci/system_integration_test.d

```diff
@@ -49,6 +52,17 @@ immutable Cleanup = [ "rm", "-rf", IntegrationPath.buildPath("node/0/.cache/"),
                       IntegrationPath.buildPath("node/6/.cache/"),
                       IntegrationPath.buildPath("node/7/.cache/"),
 ];
+auto SetGenesisTimestamp = ["sed", "-i",
+    "s/genesis_timestamp: [0-9]\\+/genesis_timestamp: curr_time/g",
```

Comment:
Wild, didn't think of this, neat.

Reply:
Well, with the exception that running this script dirties the repository, but I guess that's okay because it's in the CI script.